src/MICore/Transports/LocalLinuxTransport.cs

```diff
@@ -95,15 +99,19 @@ public override void InitStreams(LaunchOptions options, out StreamReader reader,
                     debuggeeDir = "/";
             }
 
-            string gdbStdInName = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
-            string gdbStdOutName = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+            _gdbStdInName = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+            _gdbStdOutName = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+
+            MakeGdbFifo(_gdbStdInName);
+            MakeGdbFifo(_gdbStdOutName);
 
-            MakeGdbFifo(gdbStdInName);
-            MakeGdbFifo(gdbStdOutName);
+            _fifoWatcher = new FileSystemWatcher(Path.GetTempPath());
```

Comment:
Can we specify a file path instead of a directory path? Or stick a prefix on our file? Watching for any delete for any file under /tmp seems wasteful.